ct_fe/ct_server/main.go

```diff
@@ -1,22 +1,59 @@
 package main
 
 import (
-	"fmt"
+	"errors"
 	"flag"
+	"fmt"
+	"io/ioutil"
 	"net/http"
 
 	"github.com/golang/glog"
 	"github.com/google/trillian"
+	"github.com/google/trillian/crypto"
 	"github.com/google/trillian/ct_fe"
 	"google.golang.org/grpc"
 )
 
 var rpcBackendFlag = flag.String("log_rpc_backend", "localhost:8090", "Backend Log RPC server to use")
 var serverPortFlag = flag.Int("port", 8091, "Port to serve CT log requests on")
+var trustedRootPEMFlag = flag.String("trusted_roots", "", "File containing one or more concatenated trusted root certs in PEM format")
+
+func loadTrustedRoots(filename string) (*crypto.PEMCertPool, error) {
+	if len(filename) == 0 {
+		return nil, errors.New("The --trusted_roots flag must be set to reference a valid PEM file")
```

Comment:
maybe this func should just access the flag directly rather than taking a string?
Alternatively this error text probably needs to change :)